addons/sale_timesheet/tests/test_project.py

```diff
@@ -93,3 +93,36 @@ def test_fetch_sale_order_items(self):
         self.project_global.allow_billable = False
         self.assertFalse(self.project_global._get_sale_order_items())
         self.assertFalse(self.project_global._get_sale_orders())
+
+    def test_employee_map_cost(self):
+        self.project_global.write({
+            'sale_line_id': self.so.order_line[0].id,
+        })
```

Comment:
i asked on discord about the difference between setUpClass and setUp because i putted cls.so in setUpClass, it wasn't know and then i found that it in setUp 
` @classmethod
    def setUpClass(cls, chart_template_ref=None):
        super().setUpClass(chart_template_ref=chart_template_ref)
        cls.project_global.write({
            'sale_line_id': cls.so.order_line[0].id,
        })`

Reply:
Since the so is defined in the setUp method then you could define a setUp method instead.